### ISSUE TITLE
RemovedInDjango41Warning

### ISSUE THREAD

**Issue by JulienPalard** [deschler/django-modeltranslation#596]
Got this one today using Django 3.2.2:

```
RemovedInDjango41Warning: 'modeltranslation' defines default_app_config = 'modeltranslation.apps.ModeltranslationConfig'. Django now detects this configuration automatically. You can remove default_app_config.
```

**Comment by last-partizan** [comment]
This is fine, i guess. As long as we support older versions.

**Comment by BSVogler** [comment]
So what is the plan here? To support deprecated versions as long as possible? This will block projects that are using this lib from updating to 4.1.

**Comment by last-partizan** [comment]
Plan is to support django 4.x, but not break anything older.

And this deprecation was only warning, it still works fine on django 4, even if this setting is present. Or it doesn't work on django 4?

**Comment by SaturnFromTitan** [comment]
I think there's still value in resolving this warning, even if it doesn't do direct harm:

When there are a lot of irrelevant warnings, you get blind to the important ones. So it'd be great to get this one resolved.

Probably we could check the used Django version and only set `default_app_config` when it's required. Wdyt @last-partizan? I'd be happy to contribute a PR if you agree

**Comment by last-partizan** [comment]
Sure, people are already working on this: https://github.com/deschler/django-modeltranslation/pull/631

But we need to refactor `get_version` for setup.py before we can merge it.

**Comment by last-partizan** [comment]
Should be fixed in v0.18.0.
